tests/impact/client/entities/test_project.py

```diff
@@ -0,0 +1,72 @@
+from tests.impact.client.fixtures import *
+from tests.impact.client.helpers import (
+    create_project_content_entity,
+    create_project_entity,
+)
+from modelon.impact.client import ContentType
+from tests.files.paths import SINGLE_FILE_LIBRARY_PATH
+
+
+class TestProject:
+    def test_get_project_contents(self, project):
+        contents = project.entity.get_contents()
+        assert contents == [
+            create_project_content_entity(
+                project_id="bf1e2f2a2fd55dcfd844bc1f252528f707254425",
```

Comment:
It seems like this magic ID is pasted in a lot of places. It would be nice to only have this string defined in one single place to make the test code more maintainable.